src/pptr-testing-library.ts

```diff
@@ -216,3 +216,95 @@ export const getQueriesForElement = (
 
   return queries;
 };
+
+let waitForCounter = 0;
+
+export interface WaitForOptions {
+  /**
+   * The element watched by the MutationObserver which,
+   * when it or its descendants change,
+   * causes the callback to run again (regardless of the interval).
+   * Default: `document.documentElement` (root element)
+   */
+  container?: ElementHandle;
+  /**
+   * The amount of time (milliseconds) that will pass before waitFor "gives up" and throws whatever the callback threw.
+   * Default: 1000ms
+   */
+  timeout?: number;
+  /**
+   * The maximum amount of time (milliseconds) that will pass between each run of the callback.
+   * If the MutationObserver notices a DOM change before this interval triggers,
+   * the callback will run again immediately.
+   * Default: 50ms
```

Comment:
Are we actually setting these defaults  somewhere or are these just the defaults from testing library?

If they're from testing library are we worried at all that are docs could be wrong if they change those defaults? Maybe we should set these defaults to make sure the docs and implementation stay in sync?

Reply:
These are the defaults from testing library.
Whenever there are DTL updates, I check through their changes to see if there is anything on our end we need to update. I agree that I don't want our docs to diverge from the functionality, but I also don't want our functionality to diverge from DTL's functionality. Whether we set the defaults in code here or not, we'd still have to keep an eye out to prevent it from diverging